src/VisualStudio/NuGet.Packaging.VisualStudio/Commands/CreateNuGetPackageCommand.cs

```diff
@@ -53,6 +53,7 @@ protected override void Execute()
 				{
 					// Provide default values for required fields/properties
 					viewModel.PackageId = project.Name;
+					viewModel.PackageVersion = "1.0.0.0";
```

Comment:
Shouldn't this fetch the actual version from the project, rather than hardcode a 1.0.0 every time?

Also, the default should be `1.0.0`. SemVer doesn't allow 4 segments for version :)